testing/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/DocumentationUtils.java

```diff
@@ -16,13 +16,19 @@
 
 package org.gradle.integtests.fixtures.executer;
 
+import org.gradle.util.GradleVersion;
+
 import static org.gradle.api.internal.DocumentationRegistry.BASE_URL;
+import static org.gradle.api.internal.DocumentationRegistry.BASE_URL_WITHOUT_VERSION;
 
 public class DocumentationUtils {
-    public static final String DOCS_GRADLE_ORG = "https://docs.gradle.org/";
-    public static final String PATTERN = DOCS_GRADLE_ORG + "current/";
+    public static final String PATTERN = BASE_URL_WITHOUT_VERSION + "current/";
```

Comment:
💭 `PATTERN` is a pretty bad name here, even if this was a `Pattern`.  `DOCS_CURRENT_URL` or something?